lib/ror_push_notifications/version.rb

```diff
@@ -1,3 +1,3 @@
 module Rpn
-  VERSION = '1.0.0'
+  VERSION = '2.0.0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.